### ISSUE TITLE
focal_loss_multi_class

### ISSUE THREAD

**Issue by web199195** [maozezhong/focal_loss_multi_class#1]
firstly, you should get a list which contains each class number, like classes_nu=[1,2,3] means index_0 class have 1 pic, index_1 class have 1 pics.
then, use the focal loss function like below:
model.compile(optimizer=SGD(lr=learning_rate, momentum=0.9), loss=[focal_loss(classes_num)], metrics=['accuracy'])

dear maozezhong:
classes_nu=[1,2,3] means index_0 class have 1 pic, index_1 class have 1 pics.
or classes_nu=[1,2,3] means index_0 class have 1 pic, index_1 class have 2 pics, index_2 class have 3 pics?

**Comment by maozezhong** [comment]
classes_nu=[1,2,3] means index_0 class have 1 pic, index_1 class have 2 pics, index_2 class have 3 pics

**Comment by maozezhong** [comment]
I am just lazy... I will fix it for that it will be confusing

**Comment by web199195** [comment]
thank you for your work,best wish to you !
